lib/beaker/hypervisor/aws_sdk.rb

```diff
@@ -405,7 +400,12 @@ def launch_all_nodes
           instance = create_instance(host, ami_spec, nil)
           instances.push({:instance => instance, :host => host})
         end
+        # Add metadata tags to each instance
```

Comment:
@tvpartytonight The change is that the tagging comes before the "ok" check, not after (which is what takes 20 minutes).

For v2, what restriction do you speak of? The tags cannot be passed in the v1 of the api on instantiation, only in v2, and we are using v1 currently.

(this change is a ci-blocker for f5 module, also).